core/src/main/resources/i18n/displayStrings.properties

```
@@ -605,21 +613,21 @@ portfolio.pending.step2_buyer.confirmStart.yes=Yes, I have started the payment
 portfolio.pending.step2_seller.waitPayment.headline=Wait for payment
 portfolio.pending.step2_seller.f2fInfo.headline=Buyer's contact information
 portfolio.pending.step2_seller.waitPayment.msg=The deposit transaction has at least one blockchain confirmation.\nYou need to wait until the BTC buyer starts the {0} payment.
-portfolio.pending.step2_seller.warn=The BTC buyer still has not done the {0} payment.\nYou need to wait until they have started the payment.\nIf the trade has not been completed on {1} the arbitrator will investigate.
-portfolio.pending.step2_seller.openForDispute=The BTC buyer has not started his payment!\nThe max. allowed period for the trade has elapsed.\nYou can wait longer and give the trading peer more time or contact the arbitrator for opening a dispute.
+portfolio.pending.step2_seller.warn=The BTC buyer still has not done the {0} payment.\nYou need to wait until they have started the payment.\nThe trade has to be completed until {1}.
```

Is {1} the date by which the trade needs to be completed by? If so, I suggest this.

```suggestion
portfolio.pending.step2_seller.warn=The BTC buyer still has not done the {0} payment.\nYou need to wait until they have started the payment.\nThe trade has to be completed by {1}.
```